src/dstack/api/_public/runs.py

```diff
@@ -224,7 +223,7 @@ def logs(
                     ),
                 )
                 for log in resp.logs:
-                    yield base64.b64decode(log.message)
+                    yield log.message.encode()
```

Comment:
No need to return bytes anymore – the HTTP API change is backward incompatible anyway since it no longer returns base64-encoded bytes, so we may break the Python API just as well.

Reply:
I'd suggest that we change this in a separate PR where I want to refactor the method and have two separate methods for attached and detached logs.